### PR TITLE
Add DB audit fields and remove unused code

### PR DESCRIPTION
**Pull request by clark800**

### CODE REVIEW

**Comment by clark800** [comment]
@lumberj It looks like the docker image doesn't contain the `psql` command. I got the tests passing by not using docker for the postgres tests. Does this sound like a good approach?

File: src/lib/db.js [hunk 1]
```diff
@@ -50,9 +50,33 @@ function executeSQLPlus (sqlFilepath) {
...
+      '--port=' + (connection.port || 5432),
+      '--dbname=' + connection.database,
+      '--file=' + path.resolve(sqlFilepath)
+    ]
+    const env = {
+      PATH: process.env.PATH,
+      PGPASSWORD: connection.password
+    }
+    const childProcess = spawn(command, args, {env})
+    childProcess.on('close', (code) => {
```
Comment (lumberj) [inline src/lib/db.js]: Does there need to be an `on('error')` handler?

**Comment by lumberj** [comment]
LGTM

File: src/lib/db.js (hunk 1 above)
Comment (clark800) [inline src/lib/db.js]: Yeah, I'll add that

File: scripts/ci-test.sh [hunk 2]
```diff
@@ -29,7 +29,7 @@ mysqltest() {
 postgrestest() {
   psql -U ubuntu -c 'DROP DATABASE circle_test;'
   psql -U ubuntu -c 'CREATE DATABASE circle_test;'
-  docker run --name=ledger-test-postgres -it --net=host -e LEDGER_UNIT_DB_URI=postgres://ubuntu@localhost/circle_test interledger/five-bells-ledger npm test
+  LEDGER_UNIT_DB_URI=postgres://ubuntu@localhost/circle_test npm test
```
Comment (MatthewPhinney) [inline scripts/ci-test.sh]: Why are we not using docker for this?

File: src/lib/db.js [hunk 3]
```diff
@@ -64,6 +90,8 @@ function executeScript (filename) {
 
   if (dbType === 'strong-oracle') {
     return executeSQLPlus(filepath)
+  } else if (dbType === 'pg') {
+    return executePSQL(filepath)
```
Comment (MatthewPhinney) [inline src/lib/db.js]: So now `executeStatements` is only used for sqlite? What's the advantage of doing it this way? Is this working around a `knex` issue?

File: scripts/ci-test.sh (hunk 2 above)
Comment (clark800) [inline scripts/ci-test.sh]: The docker image doesn't include the `psql` binary, which is needed

File: src/lib/db.js (hunk 3 above)
Comment (clark800) [inline src/lib/db.js]: There are scripts in the SQL file that don't work when run line by line: https://github.com/interledger/five-bells-ledger/blob/master/src/sql/pg/create.sql#L191

**Pull request merged**